### DISCUSSION TITLE
ai dock login password

### DISCUSSION THREAD

**Discussion by RICHnsk** [Avaray/stable-diffusion-templates#1] [General]
when i run this on runpod it asks for login and password, i have no idea what are those. What do i do?

**Comment by Avaray** [comment]
Default credentials (they are set in Environment Variables)
user: **user**
password: **password**

Btw. I recommend using images based on [Forge](https://github.com/lllyasviel/stable-diffusion-webui-forge).
WebUI in my current templates is much slower than Forge (about 30%?).
New templates will be available in January, probably.
